### PR TITLE
2d snowy landscape gen

### PR DESCRIPTION
**Pull request by gaba5**
https://editor.p5js.org/sebastien.lorentz/full/dVLMU3-mG

### CODE REVIEW

**Comment by gruselhaus** [comment]
@gaba5 can you give a short explanation for this change?

File: _CodingChallenges/011-perlinnoiseterrain.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ contributions:
       url: "https://mattstone.io/"
     url: "https://uploads.mattstone.io/terrain"
     source: "https://github.com/bigmstone/terrain"
-     - title: "2d snowy landscape gen"
+  - title: "2d snowy landscape gen"
     author:
       name: "Sebastien Lorentz"
       url: "https://editor.p5js.org/sebastien.lorentz/present/dVLMU3-mG"
```
Comment (GypsyDangerous) [inline _CodingChallenges/011-perlinnoiseterrain.md]: Suggest moving contribution to the bottom of the list

**Comment by gruselhaus** [comment]
Duplicate of #1718

**Comment by gaba5** [comment]
> @gaba5 can you give an short explanation for this change?

its a contributed project, a 2d terrain gen with perlin noise

**Pull request closed without merging**